src/rgw/rgw_rest_log.cc

```diff
@@ -323,20 +323,18 @@ void RGWOp_MDLog_Unlock::execute() {
 }
 
 void RGWOp_MDLog_Notify::execute() {
-  char *data;
-  int len = 0;
 #define LARGE_ENOUGH_BUF (128 * 1024)
-  int r = rgw_rest_read_all_input(s, &data, &len, LARGE_ENOUGH_BUF);
+  auto [r, data] = rgw_rest_read_all_input(s, LARGE_ENOUGH_BUF);
```

Comment:
should we use one of std::tie or c++17 structured bind consistently in the PR?  if you prefer c++17, it's easy to backport anyway

Reply:
structured bindings can be used only for local variables, that is why i have used std::tie for class member variables. I can consistently use std::tie everywhere. Let me know what you think.

Reply:
Since it's a bugfix, we may as well use std::tie.